src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ProjectUpgradesPage/ProjectUpgradesPage.tsx

```diff
@@ -1,12 +1,12 @@
 import { Trans } from '@lingui/macro'
-import { Button, Space, Statistic } from 'antd'
-import { ProjectMetadataContext } from 'contexts/projectMetadataContext'
-import { useContext, useState } from 'react'
+import { Button, Space } from 'antd'
+import { useState } from 'react'
+
 import { useIsUpgradeAvailable } from './hooks/useIsUpgradeAvailable'
 import { UpgradeWizard } from './UpgradeWizard/UpgradeWizard'
 
 export function ProjectUpgradesPage() {
-  const { cv } = useContext(ProjectMetadataContext)
+  // const { cv } = useContext(ProjectMetadataContext)
```

Comment:
**question** can probably keep this as-is, wdyt?

Reply:
oh thanks for addressing, probably shouldn't have left this page like this

`cv` got removed from ProjectMetadataContext, and we don't have a stat for "Current version" that really makes sense here (pv only goes up to 2). adding a comment for context but i'll leave this as is as a reference while this changed cv property is new